app/src/main/java/edu/drexel/lapcounter/lapcounter/frontend/LoadingActivity.java

```diff
@@ -1,21 +1,108 @@
 package edu.drexel.lapcounter.lapcounter.frontend;
 
+import android.arch.lifecycle.ViewModelProviders;
 import android.content.Intent;
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.util.Log;
 import android.view.View;
 
+import java.util.ArrayList;
+import java.util.List;
+
 import edu.drexel.lapcounter.lapcounter.R;
+import edu.drexel.lapcounter.lapcounter.backend.Database.Device.DeviceRepository;
+import edu.drexel.lapcounter.lapcounter.backend.Database.State.State;
+import edu.drexel.lapcounter.lapcounter.backend.Database.Transition.TransitionRepository;
+import edu.drexel.lapcounter.lapcounter.backend.Database.Units.Units;
+import edu.drexel.lapcounter.lapcounter.backend.Database.Workout.Workout;
+import edu.drexel.lapcounter.lapcounter.backend.Database.Workout.WorkoutRepository;
+import edu.drexel.lapcounter.lapcounter.backend.Database.Workout.WorkoutViewModel;
+import edu.drexel.lapcounter.lapcounter.backend.TimestampConverter;
 
 public class LoadingActivity extends AppCompatActivity {
 
+    /*** final static strings for units ***/
+    public final static String unitsMeters = "Meters";
```

Comment:
These static constants should be in the entities (Unit and State), not this activity.